packages/apputils/package.json

```diff
@@ -44,14 +44,14 @@
     "@phosphor/signaling": "^1.2.2",
     "@phosphor/virtualdom": "^1.1.2",
     "@phosphor/widgets": "^1.6.0",
-    "@types/react": "~16.8.8",
+    "@types/react": "^16.8.18",
```

Comment:
We have had difficulties with `@types/react` in the past, since they are the only types that we have that are listed as `dependencies` rather than `devDependencies`. Specifically, minor version bumps can cause type incompatibilities, almost by definition, and `@types/react` doesn't really follow semver because they try to keep version numbers somewhat synced to the React version.

I don't think this is wrong, necessarily, but I worry a bit about relaxing the typings, and what it might mean for extension authors that use React.

Reply:
Happy to switch this back to ~. Do you want to switch all @types back to ~, or just react?

Reply:
> since they are the only types that we have that are listed as `dependencies` rather than `devDependencies`

How about we just switch react back to ~ then.

Reply:
I'm only worried about `@types/react`, since it is listed as a real dependency. I confess that I am unsure about the ramifications of this change, I just know these typings have been challenging to match in extensions in the past.

Reply:
So, I'm not sure whether we need to change it back to `~`, I just think we should be sure to check whether this causes any downstream problems.

Reply:
Let's change it back to `~` to be safe. This PR isn't meant to make controversial decisions, just to be a routine maintenance upgrade.